src/main/java/mekanism/generators/common/MekanismGenerators.java

```diff
@@ -101,13 +102,13 @@ public void postInit(FMLPostInitializationEvent event)
 		{
 			for(IFuel s : BuildcraftFuelRegistry.fuel.getFuels())
 			{
-				if(!(s.getFluid() == null || GasRegistry.containsGas(s.getFluid().getName())))
+				if(!(s.getFluid() == null || GasRegistry.containsGas(s.getFluid().getFluid().getName())))
```

Comment:
Purely stylistic here (and I know it's not originally your code Tom), but should this be changed to `s.getFluid != null && !GasRegistry.contains...` ?

Seems a bit confusing as-is @aidancbrady